core/thread.py

```diff
@@ -859,11 +859,25 @@ def create(
         self.cache[recipient.id] = thread
 
         # Schedule thread setup for later
+        cat = self.bot.main_category
+        if len(cat.channels) == 50:
+            fallback_id = self.bot.config["fallback_category_id"]
+            fallback = discord.utils.get(cat.guild.categories, id=int(fallback_id))
```

Comment:
As `fallback_id` is `None` by default, line 865 will error. You'll need to add a `if fallback_id is not None` check from line 685 - 688:

    if fallback_id is not None:
        fallback = discord.utils.get(cat.guild.categories, id=int(fallback_id))
        if fallback and len(fallback.channels) != 50:
            self.bot.loop.create_task(thread.setup(creator=creator, category=fallback))
            return thread

Reply:
Nope, it will not error, would just return None, which is the reason I check `if fallback` in line 866

Reply:
It will still raise an error as fallback channel config variable has a `None` value. And if fallback channel variable isn't set,  `int(None)`  would anytime raise an error 😉

Reply:
o ya 😅